core/src/core/compositenode.py

```diff
@@ -36,7 +36,7 @@
 from openalea.core.metadatadict import MetaDataDict
 import logger
 
-quantify = True
+quantify = False
```

Comment:
A short comment explaining the role of "quantify" should be nice. Not sure to understand it well.